src/app/reports/reportsTable/ReportsPage.tsx

```diff
@@ -0,0 +1,130 @@
+    const [currentPage, setCurrentPage] = useState(1);
+    const startPoint = (currentPage - 1) * perPage;
+    const endPoint = currentPage * perPage - 1;
+
+    const fetchAndDisplayReportsData = useCallback(async () => {
+        setIsLoading(true);
+        if (reportTableFetchAbortController.current) {
+            reportTableFetchAbortController.current.abort("stale request");
+        }
+        reportTableFetchAbortController.current = new AbortController();
+        if (reportTableFetchAbortController.current) {
+            setErrorMessage(null);
+            const { data, error } = await getReportsDataAndCount(
+                supabase,
+                startPoint,
+                endPoint,
+                sortState,
+                reportTableFetchAbortController.current.signal
+            );
+
+            if (error) {
+                switch (error.type) {
+                    case "abortedFetchingReportsTable":
+                    case "abortedFetchingReportsTableCount":
+                        return;
+                    case "failedToFetchReportsTable":
+                    case "failedToFetchReportsTableCount":
+                        setErrorMessage(`Error occurred: ${error.type}, Log ID: 
+                    ${error.logId}`);
+                        break;
+                }
+            } else {
+                setReportsDataPortion(data.reportsData);
+                setFilteredReportCount(data.count);
+            }
+            reportTableFetchAbortController.current = null;
+            setIsLoading(false);
+            setIsLoadingForFirstTime(false);
+        }
+    }, [startPoint, endPoint, sortState]);
+
+    useEffect(() => {
+        void fetchAndDisplayReportsData();
+    }, [fetchAndDisplayReportsData]);
+
+    useEffect(() => {
+        const subscriptionChannel = supabase
+            .channel("parcels-table-changes")
+            .on(
+                "postgres_changes",
+                { event: "*", schema: "public", table: "parcels" },
+                fetchAndDisplayReportsData
+            )
+            .subscribe((status, err) => {
+                console.log(err);
+                subscriptionStatusRequiresErrorMessage(status, err, "website_data") &&
+                    setErrorMessage("Error fetching data, please reload");
+            });
+
+        return () => {
+            void supabase.removeChannel(subscriptionChannel);
+        };
+    }, [startPoint, endPoint, sortState, fetchAndDisplayReportsData]);
+
+    return (
+        <>
+            {isLoadingForFirstTime ? (
+                <Centerer>
+                    <CircularProgress aria-label="table-initial-progress-bar" />
+                </Centerer>
+            ) : (
+                <>
+                    {errorMessage && <ErrorSecondaryText>{errorMessage}</ErrorSecondaryText>}
+                    <TableSurface>
+                        <ServerPaginatedTable<ReportsTableRow, DbReportRow>
```

Comment:
Where can you style this? I think the week commencing column was a bit too narrow on smaller screens and overlapping the family header

Reply:
Can have a look into this. Though were we planning on saving styling (such as the line to demarcate the pets section) to a future ticket! Though I guess this is more important?

Reply:
I think the main point is that it should be simpler - have a look at #182 
If it's not simple let me know and we can leave it for now

Reply:
Had a look, looks doable for that one column, might require another ticket to fully polish the table?

Reply:
Done! Thanks for the suggestion, was easier than I thought!

Reply:
Have added another screen shot in this case.